apps/src/javaide/JavaIdeView.jsx

```diff
@@ -1,22 +1,132 @@
 import React from 'react';
-import AceEditor from 'react-ace';
-import 'ace-builds/src-noconflict/mode-java';
-import 'ace-builds/src-noconflict/theme-monokai';
+import JavaConsole from './JavaConsole';
+import {connect} from 'react-redux';
+import JavaEditor from './JavaEditor';
+import PaneHeader, {PaneSection} from '@cdo/apps/templates/PaneHeader';
+import {appendOutputLog} from './redux';
+import PropTypes from 'prop-types';
 
-export default class JavaIdeView extends React.Component {
-  onChange = newValue => {
-    console.log('change', newValue);
+const style = {
+  instructionsAndPreview: {
+    width: '40%',
+    display: 'block'
+  },
+  editorAndConsole: {
+    width: '60%',
+    display: 'block'
+  },
+  instructions: {
+    height: '25%',
+    marginLeft: 15,
+    marginRight: 15,
+    backgroundColor: 'white',
+    color: 'black'
+  },
+  preview: {
+    margin: 15,
+    backgroundColor: 'gray',
+    height: '200px'
+  },
+  javaIde: {
+    display: 'flex',
+    backgroundColor: '#1b1c17',
```

Comment:
Are these the official colors we'll be going with? If so, add them to the https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/color.scss file (used via @cdo/apps/util/color.js)

Reply:
they aren't official colors, but I will see if there are existing colors in there I can re-use!